server/index.js

```diff
@@ -18,63 +19,93 @@ app.use('*.js', (req, res, next) => {
 app.use(express.static('./client/dist'));
 app.use(express.json());
 
-/* ******** Personnel Routes ***************** */
+/* ******** Get Routes ***************** */
```

Comment:
I personally don't find it very necessary to have these comments separating GET, POST, etc requests.